client/templates/globals/connectioninfo.js

```diff
@@ -16,5 +24,26 @@ Template.connectionInfo.events({
         evt.preventDefault();
         console.log("Trying to reconnect...");
         Meteor.reconnect();
+    },
+
+    "click #btnWarningExpandCollapse": function (evt) {
+        evt.preventDefault();
+        let warningMessage = document.getElementById("warningMessage");
+        if(warningMessage.style.display == "none") {
```

Comment:
Again, I would prefer to make a strict equal comparison with the triple-equals operator. But this is now obsolete as well as the following comment, if you replace this whole block...